app/commands/v2/represent_downstream.rb

```diff
@@ -0,0 +1,49 @@
+module Commands
+  module V2
+    class RepresentDownstream
+      def self.name
+        self.to_s
+      end
+
+      def call(scope)
+        filter = ContentItemFilter.new(scope: scope)
+
+        draft_content_items(filter).find_each do |draft_content_item|
+          send_to_content_store(draft_content_item, Adapters::DraftContentStore)
+        end
+
+        live_content_items(filter).find_each do |live_content_item|
+          send_to_content_store(live_content_item, Adapters::ContentStore)
```

Comment:
Should we not also re-send live content items to the draft content-store? (If so this should probably happen before the draft_content_items loop.)